src/account.cpp

```diff
@@ -357,20 +364,60 @@ std::size_t olm::Account::get_prekey_json(
 void olm::Account::forget_old_prekey(
 ) {
     olm::unset(&prev_prekey, sizeof(prev_prekey));
+    num_prekeys--;
```

Comment:
What happens if this is called when there is only one prekey? In that case, I think we would want to avoid decrementing this variable